Common/Field/src/MagneticField.cxx

```diff
@@ -538,10 +538,10 @@ MagneticField *MagneticField::createFieldMap(Float_t l3Cur, Float_t diCur, Int_t
   MagFieldParam::BeamType_t btype = MagFieldParam::kNoBeamField;
   TString btypestr = beamtype;
   btypestr.ToLower();
-  TPRegexp protonBeam("(proton|p)\\s*-?\\s*\\1");
-  TPRegexp ionBeam("(lead|pb|ion|a|A)\\s*-?\\s*\\1");
-  TPRegexp protonionBeam("(proton|p)\\s*-?\\s*(lead|pb|ion|a|A)");
-  TPRegexp ionprotonBeam("(lead|pb|ion|a|A)\\s*-?\\s*(proton|p)");
+  TPRegexp protonBeam(R"((proton|p)\s*-?\s*\1)");
+  TPRegexp ionBeam(R"((lead|pb|ion|a|A)\s*-?\s*\1)");
+  TPRegexp protonionBeam(R"((proton|p)\s*-?\s*(lead|pb|ion|a|A))");
+  TPRegexp ionprotonBeam(R"((lead|pb|ion|a|A)\s*-?\s*(proton|p))");
   if (btypestr.Contains(ionBeam)) {
```

Comment:
Actually no, it's for raw strings.

https://clang.llvm.org/extra/clang-tidy/checks/modernize-raw-string-literal.html

The issue here is that `\<something>` is potentially some special character which could be swallowed, rather than used in the regular expression. It's basically avoiding you to use `\\s` which would be the unambiguous way of doing it.